dcbr-api/email_service/tasks.py

```diff
@@ -125,3 +125,18 @@ def send_registration_email(context: dict):
             context["operator"]["email_address"]
         )
     )
+
+
+@background()
+def update_registration_status():
+    expired_registrations = Registration.objects.filter(
+        expiry_date__lte=datetime.date.today(), operator_status=Registration.ACTIVE
```

Comment:
I believe we should use the current day at the end of day (23:59:59), otherwise registrations for which the timestamp is one second off in the future (but still "today") compared to the task execution time won't be updated. Please see `send_reminder_email` for an example showing how to calculate it.

Since we are now storing the expiry_date, could you please update the `send_reminder_email` task to use it rather than calculating it from scratch? This is for consistency. Please note that you will still need to apply the relativedelta to match `REMINDER_EMAIL_NOTICE_MONTHS`.

Reply:
sounds good

Reply:
This will set to `CANCELLED` the registrations expiring yesterday. If we want to expire the registrations the have expiry date set to today, we will need to set the time in the date we use for comparison to `23:59:59`.